### ISSUE TITLE
BUG: 1Password SSH agent not working

### ISSUE THREAD

**Issue by farski** [beekeeper-studio/beekeeper-studio#1344]
**Describe the bug**
I've recently started using the SSH Agent that 1Password introduced in version 8. When selecting `SSH Agent` as the option for SSH Authentication with an SSH Tunnel, I receive the error `All configured authentication methods failed`
 
**Version Information (please complete the following information):**
 - OS: macOS
 - App Version 3.6.2

**Comment by elsbrock** [comment]
This may be the reason: https://developer.1password.com/docs/ssh/agent/advanced#ssh-server-six-key-limit

**Comment by farski** [comment]
I don't believe that's the issue for me, since these keys work via 1Paassword when connecting to the same DBs with clients other than Beekeeper.

**Comment by elsbrock** [comment]
Beekeeper may not be aware of the SSH agent. Can you check this? https://developer.1password.com/docs/ssh/agent/compatibility#configure-ssh_auth_sock-globally-for-every-client

**Comment by farski** [comment]
I'm having some trouble with the launch config, but opening Beekeeper from the command line with `open -a` after exporting `SSH_AUTH_SOCK` does seem to be working. Need to dig into it some more when I have time.

**Comment by belaarany** [comment]
I have the same issue on Windows 11. I checked the links mentioned above but there are no solution for Windows. Anyone may have a workaround for Windows?

**Comment by farski** [comment]
Closing the loop: both methods suggested in [the docs](https://developer.1password.com/docs/ssh/agent/compatibility/#configure-ssh_auth_sock-globally-for-every-client) (`open -a` and creating the launch agent) do work.

I think ideally it would be great if Beekeeper natively supported `IdentityAgent` in `~/.ssh/config` so that they weren't necessary, but that is probably better classified as a feature request than a bug.

**Comment by joeld-win** [comment]
Reopening the loop, there are no methods that work on Windows.

**Comment by DKhalil** [comment]
Might be related to #1444

**Comment by rathboma** [comment]
Hey folks,

Thanks for the heads up on this. I consider this a bug, not a feature. Thanks for the workarounds. I'll add it to the official bug backlog.

**Comment by lode** [comment]
I'm not sure if this is the same, but I got it working for SSH tunnel by following these steps: https://developer.1password.com/docs/ssh/agent/compatibility/#ssh-auth-sock ("Configure `SSH_AUTH_SOCK` globally for every client").

I had to use the IP address and username, couldn't use the host name I configured in `~/.ssh/config`, but otherwise it works fine.

<img width="656" height="444" alt="Image" src="https://github.com/user-attachments/assets/2d572ad8-a841-4ed6-8c37-afccb7284259" />
